index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My portfolio</title>
+	<link rel="shortcut icon" type="image/jpg" href="images/favicon.png"/>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="normalize.css" />
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="portfoliostyle.css" />
+</head>
+<body>
+	<nav>
+		<h1>William Thornhill</h1>
+		
+		<div class="nav-buttons">
+			<button type="button">About me</button>
+		
+			<button type="button">Portfolio</button>
+		
+			<button type="button">Contacts</button>
+		</div>
+	</nav>
+	
+	<main>
+	<div class="about-projects">
+		<div class="about">
+			<h2>About me</h2>
+			<div>
+				<img src="images/dog-portrait.jpg" alt="Profile picture of dog" />
+				
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est.</p>
+			</div>
+		</div>
+		
+		<div class="projects">
+			<h2>Projects</h2>
+			<div class="project-container">
+				<div>
+					<h3>Project 1</h3>
+					
+					<img src="images/placeholder-square.jpeg" alt="Placeholder image for project thumbnail" />
+					
+					<button type="button">GitHub Link</button>
+				</div>
+			
+				<div>
+					<h3>Project 2</h3>
+					
+					<img src="images/placeholder-square.jpeg" alt="Placeholder image for project thumbnail" />
+					
+					<button type="button">GitHub Link</button>
+				</div>
+			
+				<div>
+					<h3>Project 3</h3>
+					
+					<img src="images/placeholder-square.jpeg" alt="Placeholder image for project thumbnail" />
+					
+					<button type="button">GitHub Link</button>
+				</div>
+			</div>	
+			
+		</div>
+	</div>
+		<div class="contacts">
+			<h2>Contact me</h2>
+		</div>
+		
+		<div class="contacts-icons">
+			<a href="#" class="fa fa-envelope-square"></a>
+		
+			<a href="#" class="fa fa-linkedin"></a>
+			
+			<a href="#" class="fa fa-phone"></a>
+			
+			<a href="#" class="fa fa-github-square"></a>
+			
```

Comment:
Can remove some whitespace to condense your code a little :)

Reply:
Removed unnecessary white space from index.html